airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -667,6 +671,23 @@ def execute(self, context: Context) -> dict:
                 raise
             self.log.info("Cluster already exists.")
             cluster = self._get_cluster(hook)
+        except AirflowException as ae:
+            # There still could be a cluster created here in an ERROR state which
+            # should be deleted immediately rather than consuming another retry attempt
+            # (assuming delete_on_error is true (default))
+            # This reduces overall the number of task attempts from 3 to 2 to successful cluster creation
+            # assuming the underlying GCE issues have resolved within that window. Users can configure
+            # a higher number of retry attempts in powers of two with 30s-60s wait interval
+            try:
+                cluster = self._get_cluster(hook)
+                # redundant condition checking in order to reuse _handle_error_sate
+                if cluster.status.state == cluster.status.State.ERROR:
+                    self._handle_error_state(hook, cluster)
+            except AirflowException:
+                # We could get any number of failures here, including cluster not found and we
+                # can just ignore to ensure we surface the original cluster create failure
+                pass
```

Comment:
Can we log the exception rather than just swallowing it? There is some potentially useful information here such as a cluster being stuck in the deleting state which it might be worth communicating to users.